policy/modules/services/colord.te

```diff
@@ -133,7 +133,6 @@ optional_policy(`
 ')
 
 optional_policy(`
-	udev_read_db(colord_t)
```

Comment:
Why was this removed, but the others switched over to the pid one?

Same question for consolekit below.

Reply:
The call to `udev_read_pid_files(colord_t)` is just bellow :)